doc/client_lua_api.txt

```diff
@@ -1025,6 +1025,17 @@ Methods:
     * returns last look vertical angle
 * `get_key_pressed()`:
     * returns last key typed by the player
+* `hud_add(definition)`
+    * add a HUD element described by HUD def, returns ID number on success and `nil` on failure.
+    * See [`HUD definition`](#hud-definition-hud_add-hud_get)
+* `hud_get(id)`
+    * returns the [`definition`](#hud-definition-hud_add-hud_get) of the HUD with that ID number or `nil`, if non-existent.
```

Comment:
This link may not work in a few markdown viewers due the parentheses and code expressions in the title

Reply:
done